docs/roman_datamodels/datamodels/using_datamodels.rst

```diff
@@ -94,3 +94,42 @@ page::
 	42
 	>>> dm2.meta.exposure_start_time_mjd
 	60000.0
+
+
+.. note::
+
+	All datamodels have built in validation against the defining schemas.
+	This means that if you try to assign a value that is not allowed according
+	to one of these schemas, you will get an error. This is a good thing!
+
+	If you are getting validation errors consult the corresponding schema in
+	``rad`` to se what is allowed. If you think the schema is wrong, or you
+	continue to have issues, please contact the Roman team for help.
+
+	As a method of last resort, if you wish to turn off validation, you can do
+	so by setting the environment variable ``ROMAN_VALIDATE`` to false. This is
+	not recommended!
+
+	.. code-block:: bash
+
+		export ROMAN_VALIDATE=false
+
+	To restore validation, set the environment variable to ``true`` or unset it.
+
+
+.. warning::
+
+	We strongly recommend against ever turning off validation. This can lead to
+	a variety of unrecoverable problems. Such as not being able to write out
+	your datamodel or not being able to read it back in. Or worse, the data in
+	the datamodel may not be compatible with operations intended to run on that
+	datamodel. The Roman team will not assist with fixing such problems which
+	occur when validation is turned off.
+
+	Use this feature at your own risk!
+
+	You have been warned!
```

Comment:
```suggestion
```

Reply:
It may be just me but this sounds too much.

Reply:
Using this "feature" is extremely ill advised, I want to leave no doubts with regards to this. I want it very very clear that we will not support solving any issues that arise when using this "feature", but will support solving issues that might lead to invoking this "feature" to start with.

I guess it is over the top. It raises a big warning every single time validation is ignored. So I will remove it.